projects/kserve/testing/config.yaml

```
@@ -420,6 +426,10 @@ tests:
       concurrency: 16
       plugin: caikit_client_plugin
       interface: grpc
+    longevity:
+      enabled: false
+      repeat: 3
+      delay: 300 # 5min, support "5min" in future
```

can you change this to:
```suggestion
      repeat: 2
      delay: 30 # seconds
```
(I try to keep the default values as light tests)